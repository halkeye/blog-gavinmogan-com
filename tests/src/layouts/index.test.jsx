/* eslint-env jest */
import React from 'react';
import { shallow } from 'enzyme';
import MainLayout from '../../../src/layouts/index.jsx';
// eslint-disable-next-line react/display-name
jest.mock('../../../src/layouts/Headers.jsx', () => () => <div />);
// eslint-disable-next-line react/display-name
jest.mock('../../../src/components/ProfileImage/ProfileImage.jsx', () => () => <div />);

describe('layout', () => {
  test('Renders', () => {
    const wrapper = shallow(
      <MainLayout title="hi there"><div /></MainLayout>
    );
    expect(wrapper).toBeTruthy();
    expect(wrapper.find('Navigation').props()).toMatchObject({ LocalTitle: 'hi there' });
  });
});
