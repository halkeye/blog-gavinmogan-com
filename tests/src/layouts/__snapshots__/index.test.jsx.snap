// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout Renders 1`] = `
<MainLayout
  classes={
    Object {
      "grow": "MainLayout-grow-2",
      "list": "MainLayout-list-4",
      "menuButton": "MainLayout-menuButton-3",
      "root": "MainLayout-root-1",
    }
  }
  title="hi there"
>
  <div />
</MainLayout>
`;
